src/Features/VisualBasic/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.vb

```diff
@@ -332,7 +332,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
 
         Private Function GetAttributes(tagName As String, attributes As SyntaxList(Of XmlNodeSyntax)) As IEnumerable(Of CompletionItem)
             Dim existingAttributeNames = attributes.Select(AddressOf GetAttributeName).WhereNotNull().ToSet()
-            Return GetAttributeItems(tagName, existingAttributeNames)
+            Return GetAttributeItems(tagName, existingAttributeNames, false)
```

Comment:
named parameter for `false`

Reply:
can you add a test in CompletionCommandHandlerTests that shows the vb behavior?

Reply:
Like I said in the pr, it doesn't work the same way. It more behaves like I described with my other comment, but even IF you typed "l" here.

So I could add a test, but I am not sure it would be useful.

Reply:
the test still shows behavior, and let's us know if it changes :)

code changes fairly regularly.  and if we only have tests for what we think the code itself does, we can often miss stuff if a large refactoring happens later.

Reply:
Turns out after retrying, I found out I didn't checked this well and it behaves the same.

So I ended up implementing the same logic, but still added the test to make sure.